configs/acra-server.yaml

```diff
@@ -131,6 +131,9 @@ tls_client_ca:
 # Path to server TLS certificate presented to AcraConnectors (overrides "tls_cert")
 tls_client_cert: 
 
+# Use certificate info as clientID for encryption/decryption when used. Use certificate from connector's connection if acraconnector_tls_transport_enable used or from client's connection if acraconnector_transport_encryption_disable used
```

Comment:
Interesting that we try to remove `client_id` setting and add two configurations instead

Reply:
> Interesting that we try to remove client_id setting and add two configurations instead

I guess in the future they will be enabled by default (making them opt-out, rather than opt-in). That way it will become zero configuration to most users.

Reply:
actually we will not remove this parameter because it will allow to use acra-server instead acra-connector as close to app encryption proxy. for example if app can't encrypt data itself and client wants to setup acra-server close to app on the same node/container/machine and transparently encrypt data for "e2e" without app modification. For that they can run acra-server with `--client_id` parameter and with limited key storage they can do that)